sst/packages/functions/src/cron-jobs/api-key-expiry/main.ts

```diff
@@ -0,0 +1,93 @@
+import { eq, sql } from 'drizzle-orm'
+import { Config } from 'sst/node/config'
+
+import PostmanDbClient from '@/core/database/client'
+import { apiKeys } from '@/core/models'
+import { users } from '@/core/models/user'
+import { getFutureUTCDate } from '@/core/util/date'
+import { sendEmail } from '@/core/util/email'
+
+import { IS_LOCAL, LOCAL_DB_URI } from '../../env'
+
+import { reminderEmailMapper } from './helper'
+
+/*
+- This function is triggered by a cron job that runs every day at 12AM UTC, i.e. 8AM SGT
+- It picks up API keys that are expiring 28 days, 14 days, 3 days and 1 day from now
+- DATE_TRUNC("day", valid_until) function round down the valid_until timestamp to the nearest day
+- The cron job will send emails to the API key owners to remind them to renew their API keys
+*/
+
+export async function handler() {
+  const dbUri = IS_LOCAL ? LOCAL_DB_URI : Config.POSTMAN_DB_URI
```

Comment:
Deployment note: Let's make sure this DB_URI is of a read-replica as we only need that

Reply:
so for [the next PR](https://github.com/opengovsg/postmangovsg/pull/2108), we actually have to UPDATE the table. 

do you want to use two different `DB_URI`s or should we just stick to one